pkg/registry/job/strategy.go

```diff
@@ -163,6 +165,23 @@ func JobToSelectableFields(job *batch.Job) fields.Set {
 	return generic.MergeFieldsSets(objectMetaFieldsSet, specificFieldsSet)
 }
 
+func CheckJob(r *registry.Store, ctx api.Context, err error, obj runtime.Object) error {
```

Comment:
I don't think it's needed for jobs. Jobs don't disappear from the list when doing `kubectl get jobs`, even when it's finished.